packages/twenty-front/src/modules/object-record/record-field/components/FieldInput.tsx

```diff
@@ -71,7 +73,13 @@ export const FieldInput = ({
       recordFieldInputScopeId={getScopeIdFromComponentId(recordFieldInputdId)}
     >
       {isFieldRelation(fieldDefinition) ? (
-        <RelationFieldInput onSubmit={onSubmit} onCancel={onCancel} />
+        isFieldRelationFromManyObjects(fieldDefinition) ? (
+          <RelationManyFieldInput
+            relationPickerScopeId={`relation-picker-${fieldDefinition.metadata.fieldName}`}
```

Comment:
shouldn't we have an unique id, maybe based on the field id? Also I believe you can use the function `getScopeIdFromComponentId` to build the right shape for a scope id